ansible/Vagrantfile

```diff
@@ -11,6 +11,22 @@ Vagrant.configure(2) do |config|
     vb.customize ["modifyvm", :id, "--memory", "2048"]
   end
 
+  config.vm.provider :openstack do |os|
+    config.ssh.username = "centos"
```

Comment:
``config.ssh.username = "centos"`` certainly break non-openstack usage. I replaced locally ``centos`` by ``vagrant`` I will try that ``node.ssh.username`` option